### DISCUSSION TITLE
Is it possible to disable the fork button on my sandbox, when I'm sharing it on live mode?

### DISCUSSION THREAD

**Discussion by Avivhdr** [codesandbox/codesandbox-client#7172] [Q&A]
I'm conducting FE interviews using a code sandbox I made with different questions.

I would like to disable the "fork" option once I'm in live mode so the candidate could not fork the sandbox.

Will this be possible at one point?

**Comment by sannek** [comment]
If you have a Team PRO subscription, you can toggle a setting that stops sandboxes from leaving that team. So you (or any editors in your team) could still fork it, but it cannot be forked outside of the team you created it in while that setting is active.

**Reply by Avivhdr** [reply]
Thanks, but I was looking for a way to share a sandbox with a stranger, while not allowing them to fork the code.
I have no problem that they look at it. but not forking it.
